lib/src/vs/conv.rs

```diff
@@ -74,6 +75,17 @@ pub fn try_u128_to_versionstamp(v: u128) -> Result<[u8; 10], Error> {
 	Ok(buf)
 }
 
+pub fn versionstamp_to_u64(vs: &Versionstamp) -> u64 {
+	(vs[0] as u64).wrapping_shl(8 * 8)
+		| (vs[1] as u64).wrapping_shl(7 * 8)
+		| (vs[2] as u64).wrapping_shl(6 * 8)
+		| (vs[3] as u64).wrapping_shl(5 * 8)
+		| (vs[4] as u64).wrapping_shl(4 * 8)
+		| (vs[5] as u64).wrapping_shl(3 * 8)
+		| (vs[6] as u64).wrapping_shl(2 * 8)
+		| (vs[7] as u64).wrapping_shl(8)
+		| (vs[8] as u64).wrapping_shl(0)
```

Comment:
This seems like it would not be portable. Doesn't this assume a certain endianness for integers? Maybe use `from_be_bytes`?
```suggestion
	u64::from_be_bytes(bytes[..8].try_into().unwrap())
```

Reply:
Oh it should have been `u64::from_ne_bytes(vs[..8].try_into().unwrap())`
`